### PR TITLE
Tweak CTabs and collision.

### PR DESCRIPTION
**Pull request by iTitus**

### CODE REVIEW

**Comment by iTitus** [comment]
bump :P

File: src/main/java/kineticrevolution/tileEntities/TileDuster.java [hunk 1]
```diff
@@ -50,20 +54,21 @@ public void onFallenUpon(Entity entity, float distance) {
...
+			progress = 0;
 			spawnParticles(30);
 			targetHeight = 1.0;
-			worldObj.setBlock(xCoord, yCoord - 1, zCoord, BlockLoader.duster, 0, 2);
+			worldObj.setBlock(xCoord, yCoord - 1, zCoord, BlockLoader.duster, 0, 3);
 			NBTTagCompound tag = new NBTTagCompound();
+			yCoord--;
 			writeToNBT(tag);
-			tag.setInteger("y", yCoord - 1);
+			yCoord++;
```
Comment (AEnterprise) [inline src/main/java/kineticrevolution/tileEntities/TileDuster.java]: why this change?

**Comment by AEnterprise** [comment]
not bad, will probably collide and overlap with local changes i already made though, also you might wana rebase

File: src/main/java/kineticrevolution/tileEntities/TileDuster.java (hunk 1 above)
Comment (iTitus) [inline src/main/java/kineticrevolution/tileEntities/TileDuster.java]: Easier and faster

Comment (AEnterprise) [inline src/main/java/kineticrevolution/tileEntities/TileDuster.java]: also kind of not needed i think to reset the y coord, the entity is getting removed anyways

**Comment by AEnterprise** [comment]
as predicted: merge conflicts, mainly due to the dynamic duster shrinking and progress stuff

**Comment by iTitus** [comment]
will do

**Comment by iTitus** [comment]
done

**Comment by AEnterprise** [comment]
can you squach?

**Comment by iTitus** [comment]
i did

**Comment by iTitus** [comment]
another bump :P
